ubtue_Brepols.js

```diff
@@ -60,17 +60,18 @@ function getSearchResults(doc, checkOnly) {
 
 function doWeb(doc, url) {
 	if (detectWeb(doc, url) === "multiple") {
-		Zotero.selectItems(getSearchResults(doc), function (items) {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
```

Comment:
Is this needed to solve the original problem?

Reply:
ok. No change here.